app/models/sections/info/movementScenario/MovementScenario.scala

```diff
@@ -230,16 +217,63 @@ object MovementScenario extends Enumerable.Implicits with Logging {
 
   }
 
+  /**
+   * emcs: certified_consignee / import_for_certified_consignee
+   */
+  case object CertifiedConsignee extends WithName("certifiedConsignee") with MovementScenario {
```

Comment:
This object's destinationType and movementType methods aren't tested in MovementScenarioSpec

Reply:
Will add those in